client/app/templates/Components/Form/_checkboxgroup.html.twig

```diff
@@ -70,6 +70,13 @@
 
                 <label class="govuk-label govuk-radios__label {{ labelClass }}" for="{{ choiceElement.vars.id }}">
                     {{ choiceElement.vars.label | trans({}, element.vars.choice_translation_domain ?: translationDomain)}}
+
+                    {% if conditional or labelDataTarget or multitoggle %}
```

Comment:
Target `multitoggleForRadio` instead of the array as you have done this for the other two var in this statement